src/lib/math.ts

```diff
@@ -73,13 +73,11 @@ export const loadMathJax = async (): Promise<boolean> => {
 export const mathifyElement = (elem: Element): void => {
   const mathjaxLoaded = window.MathJax !== undefined
   const innerText = elem.textContent
-
```

Comment:
Hmmm...no, definitely still showing up in the PR and on my branch. It may be subtle since it's just the removal of some empty lines:

```
$ git diff main -- src/lib/math.ts
diff --git a/src/lib/math.ts b/src/lib/math.ts
index 161a8f9..369216b 100644
--- a/src/lib/math.ts
+++ b/src/lib/math.ts
@@ -73,13 +73,11 @@ export const loadMathJax = async (): Promise<boolean> => {
 export const mathifyElement = (elem: Element): void => {
   const mathjaxLoaded = window.MathJax !== undefined
   const innerText = elem.textContent
-
   // Only try to mathify if content includes expected delimiters
   // including \( , $$ , or \[
   if (innerText === null || !/\\\(|\$\$|\\\[/.test(innerText)) {
     return
   }
-
   const queueMathJax = (elem: Element): void => {
     window.MathJax.Hub.Queue(['Typeset', window.MathJax.Hub, elem])
   }
```